mongodbatlas/advanced_clusters.go

```diff
@@ -0,0 +1,230 @@
+	resp, err := s.Client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	if l := root.Links; l != nil {
+		resp.Links = l
+	}
+
+	return root, resp, nil
+}
+
+// Get gets the cluster specified to {ClUSTER-NAME} from the project associated to {GROUP-ID}.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/clusters-get-one/
+func (s *AdvancedClustersServiceOp) Get(ctx context.Context, groupID, clusterName string) (*AdvancedCluster, *Response, error) {
+	if groupID == "" {
+		return nil, nil, NewArgError("groupId", "must be set")
+	}
+	if err := checkClusterNameParam(clusterName); err != nil {
+		return nil, nil, err
+	}
+
+	basePath := fmt.Sprintf(advancedClustersPath, groupID)
+	escapedEntry := url.PathEscape(clusterName)
+	path := fmt.Sprintf("%s/%s", basePath, escapedEntry)
+
+	req, err := s.Client.NewRequest(ctx, http.MethodGet, path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	root := new(AdvancedCluster)
+	resp, err := s.Client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return root, resp, err
+}
+
+// Create adds a cluster to the project associated to {GROUP-ID}.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/clusters-create-one/
+func (s *AdvancedClustersServiceOp) Create(ctx context.Context, groupID string, createRequest *AdvancedCluster) (*AdvancedCluster, *Response, error) {
+	if groupID == "" {
+		return nil, nil, NewArgError("groupId", "must be set")
+	}
+	if createRequest == nil {
+		return nil, nil, NewArgError("createRequest", "cannot be nil")
+	}
+
+	path := fmt.Sprintf(advancedClustersPath, groupID)
+
+	req, err := s.Client.NewRequest(ctx, http.MethodPost, path, createRequest)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	root := new(AdvancedCluster)
+	resp, err := s.Client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return root, resp, err
+}
+
+// Update a cluster in the project associated to {GROUP-ID}
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/clusters-modify-one/
+func (s *AdvancedClustersServiceOp) Update(ctx context.Context, groupID, clusterName string, updateRequest *AdvancedCluster) (*AdvancedCluster, *Response, error) {
+	if groupID == "" {
+		return nil, nil, NewArgError("groupId", "must be set")
+	}
+	if updateRequest == nil {
+		return nil, nil, NewArgError("updateRequest", "cannot be nil")
+	}
+
+	basePath := fmt.Sprintf(advancedClustersPath, groupID)
+	path := fmt.Sprintf("%s/%s", basePath, clusterName)
+
+	req, err := s.Client.NewRequest(ctx, http.MethodPatch, path, updateRequest)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	root := new(AdvancedCluster)
+	resp, err := s.Client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return root, resp, err
+}
+
+// Delete the cluster specified to {CLUSTER-NAME} from the project associated to {GROUP-ID}.
+func (s *AdvancedClustersServiceOp) Delete(ctx context.Context, groupID, clusterName string) (*Response, error) {
```

Comment:
This endpoint doesn’t have a documentation page yet, I have filed [DOCSP-17080](https://jira.mongodb.org/browse/DOCSP-17080) to add this

Reply:
is this different from using the "not advanced"  api call?

Reply:
No, the only difference is `v1.5`

Reply:
that may be the reason is undocumented, worth checking with the team as well as docs, the omission may be intentional